nntrainer/graph/network_graph.cpp

```diff
@@ -77,33 +78,24 @@ LayerNode &NetworkGraph::getSortedLayerNode(unsigned int ith) {
 
 void NetworkGraph::topologicalSort() {
   std::stack<LayerNode> Stack;
+  std::vector<bool> visited(num_node);
```

Comment:
std::vector<bool> visited(num_node) = {0};

Reply:
not work.